cronjobs/group-sync/cronjob.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: group-sync
+spec:
+  schedule: "0 * * * *"
```

Comment:
We might want to set the cronjob to be only executed during class registration period, maybe 3 weeks? Need to confirm that.

Reply:
That makes sense, probably easiest to just delete the cj once the registration period is over